### PR TITLE
Testing and working with fetch()

### PR DESCRIPTION
**Pull request by KidiH**
Changes made to the webapp 
- Made a servlet file. The servlet file contains additional Amharic proverbs and randomly selects one.
- Renamed the servlet file because it was necessary for it to be named the same as the function inside
- Added a function in script.js that connects the servlet and the html files. 
- Added a button to blog.html to generate new proverbs on click.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/RandomProverbServlet.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns a random proverb. */
+@WebServlet("/random-proverb")
+public final class RandomProverbServlet extends HttpServlet {
+    private List<String> proverbs;
+
+// Hardcoded list of the random proverbs
```
Comment (cshei) [inline portfolio/src/main/java/com/google/sps/servlets/RandomProverbServlet.java]: nit: indent to same level as the next line

File: portfolio/src/main/java/com/google/sps/servlets/RandomProverbServlet.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      proverbs.add(
+        "Do not blame God for having created the tiger, but thank Him for not giving it wings");
+      proverbs.add(
+        "Fifty lemons are a load for one person, but for fifty persons they are perfume.");
+      proverbs.add(
+        "It is foolhardy to start a fire just to see the flames.");
+  }
+
+  // Function to randomly reference and display the proverbs
+
```
Comment (cshei) [inline portfolio/src/main/java/com/google/sps/servlets/RandomProverbServlet.java]: nit: remove blank line

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -47,3 +47,14 @@ $(function() {
                 });
                 
             });
+
+// Function to fetch proverbs from server and pass to DOM
+
```
Comment (cshei) [inline portfolio/src/main/webapp/script.js]: nit: remove blank line

**Review by cshei — APPROVED** [review]

File: portfolio/src/main/webapp/blog.html [hunk 4]
```diff
@@ -143,10 +143,15 @@ <h5>Snow Sun</h5>
                 <span class="sr-only">Next</span>
             </a>
         </div>
+    </div>
+    <div id = "content6">
         <div id="content">
             <p>Click here to get a random Ethiopian proverb:</p>
             <button onclick="addRandomGreeting()">Proverb</button>
             <div id="greeting-container"></div>
+             <p>More proverbs</p>
```
Comment (ArunChauhan) [inline portfolio/src/main/webapp/blog.html]: Indent to the same level as the surrounding lines.

**Review by ArunChauhan — APPROVED** [review]
Please address the comments before merging.

File: portfolio/src/main/webapp/blog.html (hunk 4 above)
Comment (KidiH) [reply portfolio/src/main/webapp/blog.html]: Fixed!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (KidiH) [reply portfolio/src/main/webapp/script.js]: Fixed!

File: portfolio/src/main/java/com/google/sps/servlets/RandomProverbServlet.java (hunk 2 above)
Comment (KidiH) [reply portfolio/src/main/java/com/google/sps/servlets/RandomProverbServlet.java]: Fixed!

**Pull request merged**